public/locale/en.json

```diff
@@ -2056,6 +2056,9 @@
   "optional": "Optional",
   "oral_issue_for_non_oral_nutrition_route_error": "Can be specified only if nutrition route is set to Oral",
   "ordering": "Ordering",
+  "org-type-dept": "Department",
+  "org-type-root": "Root",
+  "org-type-team": "Team",
```

Comment:
why duplicate things? 🤔 

this already exists!

check: `facility_organization_type__dept`